src/app/footer/footer.component.html

```diff
@@ -143,6 +143,11 @@
                 <small>Sitemap</small>
               </a>
             </li>
+            <li class="m-0">
+              <a href="https://medium.com/dockstore" target="_blank" rel="noopener noreferrer" class="footer-link">
+                <small>Dockstore Blog</small>
```

Comment:
I would lean towards the text just saying just "Blog" instead of "Dockstore Blog" -- the link is on the Dockstore web site, other similar links in the footer say "Roadmap" and "Documentation", not "Dockstore Roadmap" nor "Dockstore Documentation", etc.

But I don't feel too strongly about it.